src/base/utils/fs.cpp

```diff
@@ -518,3 +519,9 @@ QString Utils::Fs::cacheLocation()
         locationDir.mkpath(locationDir.absolutePath());
     return location;
 }
+
+QString Utils::Fs::configurationName()
+{
+    QString envConfName = QString::fromLocal8Bit(qgetenv("QBT_CONF_NAME"));
```

Comment:
It might make sense to make this variable static. However, I can't imagine qBt changing this environment variable at run time.